vnext/ReactUWP/Modules/NetworkingModule.cpp

```diff
@@ -307,9 +307,9 @@ std::future<void> NetworkingModule::NetworkingHelper::SendRequest(const std::str
           }
           else if (!formDataPart["uri"].empty()) {
             auto filePath = winrt::to_hstring(formDataPart["uri"].asString());
-            winrt::Windows::Storage::StorageFile file = co_await winrt::Windows::Storage::StorageFile::GetFileFromPathAsync(filePath);
+            winrt::Windows::Storage::StorageFile file = winrt::Windows::Storage::StorageFile::GetFileFromPathAsync(filePath).get();
             
-            auto stream = co_await file.OpenReadAsync();
+            auto stream = file.OpenReadAsync().get();
```

Comment:
might be worth adding a comment that .get is used intentionally since this is running on a worker thread. 
idk maybe it's obvious to others :)